app-misc/cmd-wrapped/cmd-wrapped-0.2.0.ebuild

```diff
@@ -0,0 +1,85 @@
+# Copyright 2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+# Autogenerated by pycargoebuild 0.13.2
+
+EAPI=8
+
+CRATES="
+	aho-corasick@1.1.2
+	android-tzdata@0.1.1
+	android_system_properties@0.1.5
+	anstream@0.6.5
+	anstyle-parse@0.2.3
+	anstyle-query@1.0.2
+	anstyle-wincon@3.0.2
+	anstyle@1.0.4
+	autocfg@1.1.0
+	bumpalo@3.14.0
+	cc@1.0.83
+	cfg-if@1.0.0
+	chrono@0.4.31
+	clap@4.4.13
+	clap_builder@4.4.12
+	clap_lex@0.6.0
+	colorchoice@1.0.0
+	colored@2.1.0
+	core-foundation-sys@0.8.6
+	iana-time-zone-haiku@0.1.2
+	iana-time-zone@0.1.58
+	js-sys@0.3.66
+	lazy_static@1.4.0
+	libc@0.2.151
+	log@0.4.20
+	memchr@2.7.1
+	num-traits@0.2.17
+	once_cell@1.19.0
+	proc-macro2@1.0.71
+	quote@1.0.33
+	regex-automata@0.4.3
+	regex-syntax@0.8.2
+	regex@1.10.2
+	strsim@0.10.0
+	syn@2.0.43
+	unicode-ident@1.0.12
+	utf8parse@0.2.1
+	wasm-bindgen-backend@0.2.89
+	wasm-bindgen-macro-support@0.2.89
+	wasm-bindgen-macro@0.2.89
+	wasm-bindgen-shared@0.2.89
+	wasm-bindgen@0.2.89
+	windows-core@0.51.1
+	windows-sys@0.48.0
+	windows-sys@0.52.0
+	windows-targets@0.48.5
+	windows-targets@0.52.0
+	windows_aarch64_gnullvm@0.48.5
+	windows_aarch64_gnullvm@0.52.0
+	windows_aarch64_msvc@0.48.5
+	windows_aarch64_msvc@0.52.0
+	windows_i686_gnu@0.48.5
+	windows_i686_gnu@0.52.0
+	windows_i686_msvc@0.48.5
+	windows_i686_msvc@0.52.0
+	windows_x86_64_gnu@0.48.5
+	windows_x86_64_gnu@0.52.0
+	windows_x86_64_gnullvm@0.48.5
+	windows_x86_64_gnullvm@0.52.0
+	windows_x86_64_msvc@0.48.5
+	windows_x86_64_msvc@0.52.0
+"
+
+inherit cargo
+
+DESCRIPTION="Find out what the past year looks like in command line!"
+HOMEPAGE="https://github.com/YiNNx/cmd-wrapped"
+SRC_URI="
+	${CARGO_CRATE_URIS}
+	https://github.com/YiNNx/cmd-wrapped/archive/refs/tags/${PV}.tar.gz -> ${P}.tar.gz
+"
+
+LICENSE="MIT"
+# Dependent crate licenses
+LICENSE+=" MIT MPL-2.0 Unicode-DFS-2016"
+SLOT="0"
+KEYWORDS="~amd64"
```

Comment:
Rust packages need to have a `QA_FLAGS_IGNORED=` variable, otherwise the Gentoo Tinderbox (QA tool) will flag this ebuild as ignoring CFLAGS;

You can add something like
`QA_FLAGS_IGNORED="usr/bin/${PN}` (if this matches the binary path of course)